### ISSUE TITLE
Image error

### ISSUE THREAD

**Issue by anshika013** [accordproject/cicero-template-library#429]
# Bug Report 🐛
I was going through the website and found this bug in footer of the website that the image is not displaying properly on the website. 

## Possible Solution
The image should be removed or it should be displayed as icon

### Desktop
Windows 11

<img width="748" alt="Screenshot 2024-10-03 221001" src="https://github.com/user-attachments/assets/f43def97-aa3a-4894-b000-7a8c9cd929c0">

**Comment by jeyasimma** [comment]
Could you assign it to me, I try to solve it.

**Comment by sanketshevkar** [comment]
I don't think this issue belongs here. This seems to be present on out main site which is a word press site I believe.

**Comment by DianaLease** [comment]
Closing as this code base does not contain the AP marketing site.
